### DISCUSSION TITLE
Motor error flag - PHASE_RESISTANCE_OUT_OF_RANGE

### DISCUSSION THREAD

**Discussion by pipipipi2002** [tinymovr/Tinymovr#304] [Q&A - Tinymovr]
Error comes out when i try to calibrate the motor, it runs for a split second and stopped. No error in the Controller section, only the motor section. Is there a way to set the motor resistance through the gui? i tried doing it but it  keeps getting overwritten. Thanks.

<img width="479" alt="image" src="https://github.com/tinymovr/Tinymovr/assets/65348889/764baafc-96ed-479d-9cf7-706467516dfd">

**Comment by pipipipi2002** [comment]
After looking through the firmware, I found out that the phase resistance range for the R5 series is between 0.01 and 1 ohms. Is there a reason why the value cant be higher?

**Reply by yconst** [reply]
Yes -- the main reason is the accuracy of phase current measurement. In a nutshell, the current sense resistors on Tinymovr R series are best suited for a current range of a few 10s of amps. This corresponds to motors with a few 100s of milliohms max phase resistance. Anything above 1 ohm is in the range of so called "gimbal motors". You can still use Tinymovr with these motors, but you will need to measure resistance and inductance externally, and closed loop current control will be disabled. 

For a better, more detailed explanation see this docs page: https://tinymovr.readthedocs.io/en/latest/hardware/gimbal.html

**Reply by pipipipi2002** [reply]
I see... In that case, I must change the motor type to GIMBAL first and then change the resistance and inductance value right? I tried to change it but it keeps getting overwritten, must be because of the motor type was wrong.

**Reply by yconst** [reply]
Exactly, if the motor is high current type, the values will be overwritten by those measured in calibration.

**Reply by pipipipi2002** [reply]
Thank you so much. Also, in gimbal mode does the IBUS gets disabled as well? I tried running the motor and it shows 0 A throughout

**Comment by yconst** [comment]
Yes, currently it is disabled as it relies on current sensing. We may revise the open loop current controller to include Id setpoints in the future though.

**Reply by pipipipi2002** [reply]
Thank you for the help. Really impressed by the can bus dfu code btw 👍

**Reply by yconst** [reply]
You're welcome! Glad to hear!
